gobblin-compaction/src/main/java/gobblin/compaction/dataset/DatasetsFinder.java

```diff
@@ -51,6 +46,7 @@
   protected final Configuration conf;
   protected final FileSystem fs;
   protected final String inputDir;
+  protected final Optional<String> alternateInputDir;
```

Comment:
Removed. Again this will be handled in li-finder. `additional` is already used in gobblin-compaction. I did not want to overload the term.